client_test.go

```diff
@@ -112,3 +134,67 @@ func ExampleClient_Query_scan() {
 	// Output:
 	// &{ID:12345678-1234-5678-1234-567812345678 Name:Alice Age:30}
 }
+
+func ExampleClient_Batch() {
+	c, err := NewStaticTokenClient(
+		endpoint, token,
+		WithDefaultKeyspace("example"),
+	)
+	if err != nil {
+		log.Fatalf("failed to initialize client: %v", err)
+	}
+
+	err = c.Batch(
```

Comment:
This is a nice syntax for this. Then if you want you want to programmatically build a batch you would create a slice?

```go
batch := make([]*Querys, 0, 0)

// Logic

batch = append(batch, c.Query(`....`)

// ...

c.Batch(BatchUnlogged, batch...)
```

Reply:
Correct